javascript-carousel/main.js

```diff
@@ -0,0 +1,129 @@
+// global dom variables
+const $leftButton = document.querySelector('.fa-chevron-left');
+const $rightButton = document.querySelector('.fa-chevron-right');
+const $buttonOne = document.querySelector('#one');
+const $buttonTwo = document.querySelector('#two');
+const $buttonThree = document.querySelector('#three');
+const $buttonFour = document.querySelector('#four');
+const $buttonFive = document.querySelector('#five');
+const $test = document.querySelectorAll('.img');
+const $buttonNodes = document.querySelectorAll('i');
+
+let count = 0;
+let countDownId = null;
+// data for entries
+
+$rightButton.addEventListener('click', function (event) {
+  if (count >= 4) {
+    count = 0;
+    viewSwap(count);
+  }
+});
+
+$leftButton.addEventListener('click', function (event) {
+  if (count <= 0) {
+    count = 4;
```

Comment:
You don't want to use hard coded values like 4, you want this to be dynamic. If you added more images to your HTML, then what happens? We want it to work with all of them